src/app/Main/Body/QueryPage/QueryPageContent/Parameter/ParameterContent-List.component.tsx

```diff
@@ -1,83 +1,22 @@
 /** @jsx jsx */
 import { css, jsx } from "@emotion/core";
-import { List, ListItem, ListItemText, ListSubheader } from "@mui/material";
-import { TFunction } from "i18next";
-import { useTranslation } from "react-i18next";
+import { List, ListSubheader } from "@mui/material";
 import { FilterType } from "../../../../../Shared/Model/Filter.model";
-import { getMicroorganismLabelService } from "../../Services/getMicroorganismLabel";
 
-const spaceStyle = css`
-    padding: 0;
-    margin: 0;
-`;
 const listStyle = css`
     padding: 0;
     margin: 0 4em 0 0;
 `;
 const listHeaderStyle = css`
     position: relative;
 `;
-const parameterValue = css`
-    margin-top: 0;
-    margin-left: 2em;
-    span {
-        letter-spacing: 0;
-    }
-`;
-
-/**
- * @desc Create a ListItem for each parameter
- * @param parameterList - list of selected parameter (filter)
- * @returns {JSX.Element[]} - list of ListItem-components
- */
-function createListItemComponent(
-    parameterAttribute: FilterType,
-    parameterList: string[],
-    t: TFunction
-): JSX.Element[] {
-    const elements: JSX.Element[] = [];
-    parameterList.forEach((parameter): void => {
-        const key = parameter.replace(".", "");
-        let parameterName: string | JSX.Element = "";
-        if (parameterAttribute === "microorganism") {
-            const translateRootString = `FilterValues.formattedMicroorganisms.${key}`;
-            const prefix = t(`${translateRootString}.prefix`);
-            const name = t(`${translateRootString}.name`);
-            const italicName = t(`${translateRootString}.italicName`);
-            const suffix = t(`${translateRootString}.suffix`);
-            parameterName = getMicroorganismLabelService({
-                prefix,
-                name,
-                italicName,
-                suffix,
-            });
-        } else {
-            parameterName = t(`FilterValues.${parameterAttribute}.${key}`);
-        }
-
-        const keyName = parameter.replace(" ", "_");
-        elements.push(
-            <ListItem key={`listItem-${keyName}`} css={spaceStyle}>
-                <ListItemText
-                    id={`labelId-${keyName}`}
-                    primary={parameterName}
-                    css={parameterValue}
-                />
-            </ListItem>
-        );
-    });
-    return elements;
-}
 
 export interface ListProps {
     parameterLabel: FilterType;
-    parameterList: string[];
+    parameterValuesList: JSX.Element[];
```

Comment:
Reject: 
First this is a layout now and you should name it accordingly.
Second: You have splitted the ``<List>`` specific elements now. You now assume that the caller of this component know that he should use ``<ListItems>`` as input components, so the encapsulation principle is broken.
Solutions:
1. If you want to behave this like a layout keep all the ``<List>`` specific elements in this components and adapt the input from the parent component accordingly.
2. Reverse your changes and let this component be a view again and use only the data this component should display as input (translated or not).

Reply:
Solution 1 only party done, see comment in corresponding commit.

Reply:
Done